apps/studio/components/interfaces/Settings/Logs/LogSelection.tsx

```diff
@@ -165,12 +197,12 @@ ${JSON.stringify(fullLog.metadata, null, 2)}
               className="cursor-pointer transition hover:text-foreground h-8 w-8 px-0 py-0 flex items-center justify-center"
               onClick={onClose}
             >
-              <IconX size={14} strokeWidth={2} className="text-foreground-lighter" />
+              <XIcon size={14} strokeWidth={2} className="text-foreground-lighter" />
             </Button>
           </div>
           <div className="h-px w-full bg-selection rounded " />
         </div>
-        {isLoading && <Loading />}
+        {isLoading || (warehouseLogQuery.isLoading && <Loading />)}
```

Comment:
looks like it should be this instead?
```suggestion
        {(isLoading || warehouseLogQuery.isLoading) && <Loading />}
```